File: apps/web/projects/identity-domain/src/lib/section.ts
/**
 * A top-level section of the web shell. Mirrors the backend `Section` enum
 * (Slice 9) — the backend decides which sections a user may see and returns the
 * set on `/bff/me`; the shell renders sidebar nav and gates routes from it.
 */
export type Section =
  | 'DASHBOARD'
  | 'ROLE_APPOINTMENT'
  | 'STRUCTURAL_ADMIN'
  | 'SABHA_DEFINITION'
  | 'OCCURRENCE_REOPEN'
  | 'SANCHALAK_PROXY';

File: apps/backend/identity-service/identity-domain/identity-domain-core/src/main/java/org/sabha/identity/domain/Section.java
package org.sabha.identity.domain;

/**
 * A top-level section of the Angular web shell (Slice 9). The set a given user
 * may see is decided by {@link VisibleSections}; later slices fill each section
 * with screens. The web app renders sidebar nav from the visible set.
 */
public enum Section {
    DASHBOARD,
    ROLE_APPOINTMENT,
    STRUCTURAL_ADMIN,
    SABHA_DEFINITION,
    OCCURRENCE_REOPEN,
    SANCHALAK_PROXY,
    SELECTION
}
